main branch test failure with test_fake_pirep

```
_______________________________ test_fake_pirep ________________________________

    def test_fake_pirep():
        """Test fake PIREP."""
        tp = Pirep(get_example_file("PIREP.txt"), utcnow=utc(2024, 7, 9, 0, 0))
>       assert tp.afos is None
E       AssertionError: assert 'PIREP' is None
E        +  where 'PIREP' = <pyiem.nws.products.pirep.Pirep object at 0x7f7e2c5f4810>.afos

tests/workflows/test_fake_afos_dump.py:17: AssertionError
```

--- a/tests/workflows/test_fake_afos_dump.py
+++ b/tests/workflows/test_fake_afos_dump.py
@@ -14,8 +14,7 @@ from pywwa.workflows import fake_afos_dump
 def test_fake_pirep():
     """Test fake PIREP."""
     tp = Pirep(get_example_file("PIREP.txt"), utcnow=utc(2024, 7, 9, 0, 0))
-    assert tp.afos is None
-    fake_afos_dump.compute_afos(tp)
+    # pyiem now sets the afos value when it is none to PIREP
     assert tp.afos == "PIREP"
 
 

